diff --git a/target/product/embedded.mk b/target/product/embedded.mk
index 55de3b9..995d57b 100644
--- a/target/product/embedded.mk
+++ b/target/product/embedded.mk
@@ -94,5 +94,4 @@ PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
 PRODUCT_COPY_FILES += \
     system/core/rootdir/init.usb.rc:root/init.usb.rc \
     system/core/rootdir/init.usb.configfs.rc:root/init.usb.configfs.rc \
-    system/core/rootdir/ueventd.rc:root/ueventd.rc \
     system/core/rootdir/etc/hosts:system/etc/hosts
